lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -207,6 +206,14 @@ def database_yaml
           nil
         end
 
+        def load_yaml(src)
+          if Psych::VERSION > '4.0'
+            YAML.safe_load(src, aliases: true)
```

Comment:
Can you use `YAML.safe_load_file('config/database.yml', aliases: true)` instead of `YAML.safe_load(src, aliases: true)`?

Reply:
I checked the sources for `psych`, and seems like `:aliases` are supported only by `safe_load`, not `safe_load_file` (which is a bit surprising to me).

Reply:
Hm, it seems that keyword args are delegated. So `:aliases` will also be delegated from `safe_load_file` to `safe_load`.
https://github.com/ruby/psych/blob/v4.0.0/lib/psych.rb#L580-L589

Reply:
Thanks! Made an update.